kolibri/plugins/management/assets/src/views/manage-content-page/channels-grid.vue

```diff
@@ -65,6 +65,13 @@
     >
       {{ $tr('deleteFailureNotification') }}
     </ui-alert>
+    <ui-alert
```

Comment:
I would be okay with using either component, but am gravitating towards UI alerts for heavy actions like channel import/content import/syncing data because they're so much more prominent than snackbars. If we want to choose one or the other though, I would wait until we standardize guidelines for them and figure out what's important for the user to see here. 

I think deletesuccess should have the same green success color importsuccess has. Since it's an indicator of a successful action, i would be reluctant to color it the same as info, warning, or error, since it's none of those.